adafruit_hid/keyboard.py

```diff
@@ -162,7 +162,8 @@ def _remove_keycode_from_report(self, keycode: int) -> None:
     @property
     def led_status(self) -> bytes:
         """Returns the last received report"""
-        return self._keyboard_device.last_received_report
+        # get_last_received_report() returns None when nothing was received
+        return self._keyboard_device.get_last_received_report() or b"\x00"
```

Comment:
The documentation says that `last_received_report` also returned `None`, so is the `or b"\00"` just a bugfix?  I'm assuming it MUST be a `bytes` object return instead of `Optional` based on usage in `led_on`.  Just wanted to check before merging so I get the semver correct.

Reply:
Yes, it makes the code compatible with the documentation. In reality `last_received_report` and `get_last_received_report()` never return None as currently implemented, but I added that to future-proof it a little in case that's fixed.
There might be bigger changes coming in 8.0.0 though, so that might be revisited soon.
See https://github.com/adafruit/circuitpython/pull/6767